modules/040-node-manager/hooks/discover_packages_proxy_addresses.go

```diff
@@ -34,7 +34,8 @@ const (
 )
 
 var _ = sdk.RegisterFunc(&go_hook.HookConfig{
-	Queue: "/modules/node-manager",
+	OnBeforeHelm: &go_hook.OrderedConfig{Order: 1},
```

Comment:
in general we use 1 in the node-manager module for migrations, so 5 will be the reliable value here